Dockerfile

```
@@ -15,6 +15,9 @@ RUN cat /etc/os-release \
     && cp -irvf bootstrap / \
     && cp -irvf manifests / \
     && cp -ivf *.repo /etc/yum.repos.d/ \
+    && rpm-ostree remove \
```

```suggestion
    && rpm-ostree override remove \
```
It appears `rpm-ostree remove` requires a live ostree system, while `rpm-ostree override remove` doesn't

---

Oh, aren't we pulling the latest build of rpm-ostree into master now? I saw https://github.com/coreos/rpm-ostree/commit/ae2f11e7a675d314041890a1eb470f6698e838a9 and assumed this would work now.

---

heh, we are but we're only replacing it a few lines below :sweat_smile: I'll go back to `override remove` for now then.